tests/test_modeling_utils.py

```diff
@@ -1428,7 +1429,7 @@ def test_safetensors_on_the_fly_wrong_user_opened_pr(self):
             bot_opened_pr_title = None
 
             for discussion in discussions:
-                if discussion.author == "SFconvertBot":
+                if discussion.author == "SFconvertbot":
```

Comment:
➕ on @julien-c's comment, have had feedback that this is not explicit enough.
```suggestion
                if discussion.author == "HuggingFaceOfficialSafetensorConverter":
```
`bot` is scary for some 😅

Reply:
we can't change the account name now

but we will think of a way to make it clearer in the UI that it's a "official bot"

Reply:
Sounds good 👍🏻